File: MealPlanner/code/src/test/java/edu/westga/cs3212/mealplanner/model/planner/TestConstructor.java
package edu.westga.cs3212.mealplanner.model.planner;

import edu.westga.cs3212.mealplanner.model.Planner;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

import java.util.ArrayList;

class TestConstructor {

	@Test
	void testParameterlessConstructor() {
		var newPlanner = new Planner();
		
		var expected = new ArrayList<String>();
		var actual = newPlanner.getPlannedMeals();
		
		assertEquals(expected, actual);
	}

}

File: MealPlanner/code/PythonServer/Test/Planner/test_constructor.py
import unittest

from Server.Data.Planner import Planner


class TestConstructor(unittest.TestCase):

    def test_constructor(self):
        newPlanner = Planner()
        expected = {}
        actual = newPlanner.plannedMeals

        self.assertDictEqual(expected, actual)

if __name__ == '__main__':
    unittest.main()
